scripts/build_helper.py

```diff
@@ -0,0 +1,30 @@
+import toml
+
+
+def modify_pyproject(package_name):
+    with open("pyproject.toml", "r") as f:
+        pyproject_data = toml.load(f)
+
+    pyproject_data["project"]["name"] = package_name
+
+    if package_name == "ragna-base":
+        pyproject_data["tool"]["setuptools"]["dynamic"]["dependencies"] = {
```

Comment:
Last time I checked, dynamic dependencies were marked as beta feature in `setuptools`. I'm ok with using this here, but could you check whether our lower limit of `setuptools` as build backend is sufficient for this to be available?

https://github.com/Quansight/ragna/blob/a45bd9063f106a7728c5fb65c9077387b25c9706/pyproject.toml#L1-L6

Reply:
I would recommend at least `setuptools>=64`, which is the version that started supporting PEP 660-based editable installations for `pyproject.toml`-style builds (PEP 517/518). Using the latest version (`v70.X`) is recommended in order to incorporate more recent bug fixes, of course, so something like `>=66` should be a reasonable margin for support.

Reply:
Sure, I'd be happy to update it to `>=66`

Reply:
Done in [`9b0f1c2`](https://github.com/Quansight/ragna/pull/405/commits/9b0f1c27f8f175e631acd771599be1a106ddf399)